From 016e34f1cfa16797d2d56f5a009f2b57e0385bd6 Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Wed, 4 Jul 2018 11:11:37 +0800
Subject: [PATCH 1335/1345] arm64: dts: marvell: add pin controls sdhci in dts
 node for admada-7040-db platform

Add pin controls in sdhci dts node of ap806 and cp110-master config
space. This patch refer to sdk kernel commit a623339a2613("arm64:
mvebu: dts: Add pin controls to SDHC entries")

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db.dts |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db.dts b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
index 01b6755..a51b84d 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
@@ -353,6 +353,8 @@
 	bus-width = <4>;
 	no-1-8-v;
 	non-removable;
+	pinctrl-names = "default";
+	pinctrl-0 = <&ap_emmc_pins>;
 };
 
 &cpm_sdhci0 {
@@ -361,6 +363,7 @@
 	no-1-8-v;
 	non-removable;
 	pinctrl-names = "default";
+	pinctrl-0 = <&cpm_sdhci_pins>;
 };
 
 &cpm_mdio {
-- 
1.7.9.5

